assets/scss/_custom.scss

```diff
@@ -354,12 +354,46 @@ main {
     word-break: break-word;
   }
 
+/* SCSS Related to the Metrics Table */
+
+    @media (max-width: 767px) { // for mobile devices, Display the names, Stability levels & types
+
+      th:nth-child(n + 4),
+      td:nth-child(n + 4) {
+        display: none;
+      }
+
+      td.metric_type{
+        min-width: 7em;
+      }
+      td.metric_stability_level{
+        min-width: 6em;
+      }
```

Comment:
lets restrict this to tables of class `metrics`

Reply:
Done, I've revised to limit the scope of changes to tables of `.metrics` class